Engine/Client/Src/ClientSceneManager/ClientSceneManager.hpp

```diff
@@ -10,6 +10,7 @@
 #include "ASceneManager.hpp"
 
 #define FIRST_SCENE "Menus/mainMenu.json"
+#define FIRST_GAME_SCENE "firstScene.json"

```

Comment:
As I mentioned it earlier, it doesn't have to be here

Reply:
Same here.